select2-parent/select2/src/main/java/org/wicketstuff/select2/Select2Choice.java

```diff
@@ -77,7 +77,13 @@ public Select2Choice(String id, ChoiceProvider<T> provider)
 		super(id, provider);
 	}
 
-	@Override
+    // needed to be able override the class on Scala
```

Comment:
In order to override a method it should be defined by the parent class. I can go around that by rolling my own class the defines onInitialize